ci/build-docs.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash -e
+
+# Map impala image to quickstart.cloudera for ibis-notebooks
+ping -c 1 impala
+
+ip=($(hostname -i impala))
+
+echo "${ip[0]} quickstart.cloudera" >> /etc/hosts
```

Comment:
You should be able to setup as an alias for impala image: https://docs.docker.com/compose/compose-file/#aliases